container_test.go

```diff
@@ -438,20 +438,6 @@ func TestProvide(t *testing.T) {
 	require.NotNil(t, p1.c1.gc1, "Grandchild1 must have been registered")
 }
 
-func TestConcurrentAccess(t *testing.T) {
```

Comment:
What I meant was this test is repeatedly Providing single constructor which is a failure with current PR since it now results in an error. Rather than adding efforts on fixing this test just to remove in the follow up PR, Im simply removing it here.